public/app/features/alerting/unified/components/RuleGroupPicker.tsx

```diff
@@ -45,6 +45,7 @@ export const RuleGroupPicker: FC<Props> = ({ value, onChange, dataSourceName })
     return [];
   }, [rulesConfig]);
 
+  // @TODO replace cascader with separate dropdowns
```

Comment:
nit: same thing here. I think it would be good to keep track of this in an issue instead of having todos in the code base since it is pretty big.

Reply:
this one is already solved in another PR